File: app/src/main/java/com/blog/demo/custom/PhoneTextViewActivity.java
package com.blog.demo.custom;

import android.app.Activity;
import android.os.Bundle;

import androidx.annotation.Nullable;

import com.blog.demo.custom.widget.PhoneTextView;

public class PhoneTextViewActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(new PhoneTextView(this));
    }

}

File: app/src/main/java/com/blog/demo/custom/PhoneTextViewActivity.kt
package com.blog.demo.custom

import android.app.Activity
import android.os.Bundle
import com.blog.demo.custom.widget.PhoneTextView

class PhoneTextViewActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(PhoneTextView(this))
    }

}